### ISSUE TITLE
Imperial format?!?

### ISSUE THREAD

**Issue by jumpjack** [CSSEGISandData/COVID-19#69]
Can you please use international format for dates? My Italian excel is getting crazy to interpret your data... We are not in 1800.

**Comment by iandoug** [comment]
I agree, but think changing it now is going to result in a lot of upset people..... they should have used program-friendly date and time formats from day 1.

**Comment by minibota** [comment]
Maybe adding a new date column for this ISO format? So that it does not break people's code...?

**Comment by cipriancraciun** [comment]
Given how many people are already using this dataset in the current format, I think such a change would break everything once more.

Moreover, in order to keep things tidy and help the JHU team, would you consider closing this issue and following #318 which discusses the same subject and seems to have the most up-votes.
